src/start.clj

```diff
@@ -40,21 +42,35 @@
                   (with-open [consumer (.createConsumer session queue)]
                     (.start connection)
                     (.receive consumer timeout))))))]
-    (with-push-to-cloud-jms-connection url queue username password receive)))
+    (with-push-to-cloud-jms-connection environment receive)))
 
 (defn plus
   "Return the sum of A and B. Just for testing."
   [^Integer a ^Integer b]
   (+ a b))
 
+(defn message-loop
+  "A blocking message loop that periodically do something."
```

Comment:
typo: "does" 
What exactly does this do though? And if it doesn't do anything interesting, should it be merged into master?

Reply:
Thanks! Yeah as you can see it's not doing anything meaningful yet. (this is a "draft" PR, so things will change before it becomes a PR ready for merge. But early feedback is always welcome)